packages/components/src/App.tsx

```diff
@@ -1,34 +1,17 @@
-import Button from './particles/Button';
+import Client from './systems/Client';
+import PharmacySearch from './systems/PharmacySearch';
 
 const App = () => {
   return (
-    <article class="prose lg:prose-lg p-4 theme-photon">
-      {' '}
-      <h1 class="mb-0">Buttons</h1>
-      <h3>Variants</h3>
-      <div class="flex flex-wrap items-center justify-left gap-2 overflow-x-hidden bg-top">
-        <Button variant="primary">Primary</Button>
-        <Button variant="secondary">Secondary</Button>
-      </div>
-      <h3>Sizes</h3>
-      <div class="flex flex-wrap items-center justify-left gap-2 overflow-x-hidden bg-top">
-        <Button variant="primary" size="xl">
-          X-Large
-        </Button>
-        <Button variant="primary" size="lg">
-          Large
-        </Button>
-        <Button variant="primary" size="md">
-          Medium
-        </Button>
-        <Button variant="primary" size="sm">
-          Small
-        </Button>
-        <Button variant="primary" size="xs">
-          Extra Small
-        </Button>
-      </div>
-    </article>
+    <Client
+      id="7N9QZujlNJHL8EIPqXpu1wq8OuXqoxKb"
+      org="org_KzSVZBQixLRkqj5d"
+      domain="auth.boson.health"
+      audience="https://api.boson.health"
+      uri="https://api.boson.health/graphql"
+    >
+      <PharmacySearch />
+    </Client>
```

Comment:
What's going on in this file? the before is also confusing to be fair

Reply:
This file is kinda like the equivalent of `/elements/index.html`, it's a non exported local app that I'm using to display some of the components.